VERSION

```diff
@@ -1 +1 @@
-0.7.0
+0.7.1
```

Comment:
Keep the newline?

Reply:
The new line was added due to a previous change on my part that was done manually, the new line should not exist.